google/ggrc-core
# GGRC-6644 Modify Control model to convert Categories attributes to strings

# Issue description

Sync service send string value for Categories attributes of Control. This value should be saved with Contol and displayed on UI. Fields that should be modified: categories, assertions.

New Json format:
categories: ["cat1", "cat2", "cat3"],
assertions: ["assertion1"]

# Steps to test the changes

* Check if api requests work proper for Control (POST/PUT/GET/DELETE). Request and result body should contain kind, means, verify_frequency fields as strings.
* Check export of Control
* Check api that is based on revisions
* Check fulltext functionality

# Solution description

Removed links between Control and Categories/Categorization models.
Added string fields "categories", "assertions" to Control.
Migrated old categories values into new string fields.
Updated old revisions in code to show categories as strings.
Modified fulltext handler for categories values.
Corrected existing tests to work with new format of Control fields.

# Sanity checklist


# Migration checklist

# PR Review checklist
